client/src/components/jobPosting/JobPostingStages.js

```diff
@@ -14,7 +14,26 @@ export class JobPostingStages extends Component {
     }
   }
 
+  componentDidMount() {
+    if (this.props.stages !== null) {
+      this.addCopiedStagesToNewJobPosting()
+    }
+  }
+
+  async addCopiedStagesToNewJobPosting() {
+    await this.props.stages.forEach((stage) => {
+      if (stage.stageName !== 'Applied' && stage.stageName !== 'Rejected' && stage.stageName !== 'Accepted') {
+        this.props.addNewStageForJobPosting({ stageName: stage.stageName, canRemove: true })
+      }
+    })
+    await this.props.clearCopiedStages()
+  }
+
   addNewStage = () => {
+    if (this.state.newStageName.length === 0 || !this.state.newStageName.trim() || this.state.newStageName.length > 255) {
```

Comment:
Tätä ei nyt tarvitse alkaa välttämättä tähän pullariin korjaamaan, mutta tulevaisuudessa näin pitkiä yksittäisiä rivejä kannattaa rivitellä. On mukavampi lukea :)